python3/kinesis/README.md

```diff
@@ -120,7 +120,7 @@ For a complete list of options, see the configuration parameters below the code
 
 ```shell
 aws cloudformation deploy
---template-file $(pwd)/cloudformation-template.output.yaml
+--template-file $(pwd)/kinesis-template.output.yaml
```

Comment:
no need for this change (or change in cloudwatch too)

Reply:
@ronish31 it's just so it will be  the same file from the previous command (there was mismatch)